### DISCUSSION TITLE
Getting FPS drops and stuttering gameplay in Multithreaded mode

### DISCUSSION THREAD

**Discussion by Modsnotworking** [ppy/osu#21917] [Q&A]
I am still getting fps drops in multithreaded mode but now I am only getting the drops in the gameplay side only. Singlethreaded mode give me a better gameplay performace but 15ms+ input latency, while mutithreaded mode gives me a much better experience(lower input latency below 8-9ms) but gives me fps drops and stuttering gameplay performace.

Logs (if of any help):
[performance.log](https://github.com/ppy/osu/files/10474228/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10474229/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10474230/updater.log)
[database.log](https://github.com/ppy/osu/files/10474231/database.log)
[input.log](https://github.com/ppy/osu/files/10474232/input.log)
[network.log](https://github.com/ppy/osu/files/10474233/network.log)

osu!(lazer) version: 2023.121.0

**Comment by peppy** [comment]
Can you provide specs of your PC/device? If it's better, it likely means you don't have enough CPUs, which while rare could be a thing.

**Reply by Modsnotworking** [reply]
Here are the specs:
Processor	Intel(R) Celeron(R) CPU  J3060  @ 1.60GHz   1.60 GHz (Dual core @1.60Ghz)
Installed RAM	4.00 GB
System type	64-bit operating system, x64-based processor

**Reply by Modsnotworking** [reply]
I have provided you with the specs. I still get fps drops and stuttering gameplay performace in multithreaded mode.

**Reply by Shirou27** [reply]
it's only dual core so that's why. i recommend you to use octa core for better performance

**Reply by Modsnotworking** [reply]
tbh I only need low input latency and only the gameplay is being affected. Only if there's a way to optimize gameplay performace...
